server/store/needs.js

```diff
@@ -18,14 +18,16 @@ const createNeed = async needDetails => {
   getVehiclesInRange({ lat: parseFloat(needDetails.pickup_latitude), long: parseFloat(needDetails.pickup_longitude) }, 7000);
 
   // Set TTL for need
-  redis.expire(`needs:${needId}`, config('needs_ttl'));
+  setNeedTTL(needId);
   return needId;
 };
 
 const deleteNeed = async needId => {
   return await redis.del(`needs:${needId}`);
 };
 
+const setNeedTTL = async needId => redis.expire(`needs:${needId}`, config('needs_ttl'));
```

Comment:
I think you can remove the `async` from this line. We don't actually have to `await` for a response from this one.